ci-operator/step-registry/load-balancer/load-balancer-commands.sh

```diff
@@ -0,0 +1,155 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ "${CONFIG_TYPE:-}" != *"externallb"* ]]; then
+    echo "CONFIG_TYPE does not contain externallb, exiting"
+    exit 0
+fi
+
+if [ ! -f "${SHARED_DIR}/LB_HOST" ]; then
+    echo "${SHARED_DIR}/LB_HOST does not exist, exiting"
+    exit 0
+fi
+
+SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey
+MASTER_IPS=$(<"${SHARED_DIR}/MASTER_IPS")
+WORKER_IPS=$(<"${SHARED_DIR}/WORKER_IPS")
+LB_USER=${LB_USER:-cloud-user}
+LB_HOST=$(<"${SHARED_DIR}/LB_HOST")
+
+if [ -f "${SHARED_DIR}/API_IP" ]; then
+    API_IP=$(<"${SHARED_DIR}/API_IP")
+fi
+
+if [ -f "${SHARED_DIR}/INGRESS_IP" ]; then
+    INGRESS_IP=$(<"${SHARED_DIR}/INGRESS_IP")
+fi
+
+# Specific to OpenStack
+if [  -f "${SHARED_DIR}/MACHINES_SUBNET_ID" ]; then
```

Comment:
Is there a way to add a guard that it's indeed an openstack job? It might have strange side effect if another platforms starts using a MACHINES_SUBNET_ID file otherwise.

Reply:
yes, we could use `OPENSTACK_CONTROLPLANE_FLAVOR` maybe?